index.html

```diff
@@ -878,14 +878,6 @@ <h2 class='informative'>Implementation Patterns</h2>
         </p>
         <ul>
           <li>Resource Identifier: the URI of the <a>LDPR</a> that was added, changed or removed.</li>
-          <li>
-            Timestamp: a timestamp marking the time of the event. The value of the timestamp
```

Comment:
Based on this [comment](https://github.com/fcrepo/fcrepo-specification/pull/58#issuecomment-281091019), I was expecting this text to now look like:

>       Timestamp: a timestamp marking the time of the event. The value of the timestamp
>       SHOULD correspond to the time the resource was added/modified/deleted, but there is no
>       strict requirement about the precision of this value.